android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -342,18 +496,65 @@ fun getPromotionalOffer() : ErrorContainer {
 
 // region private functions
 
+internal fun mapStringToProductType(type: String) : ProductType {
```

Comment:
FFTI but I'm thinking some of these functions might be more organized if we add them as extension functions... In this case, for example, we could have a `fun String.toProductType(): ProductType`. I think it's ok like this though.